lib/core/src/lib/form/components/widgets/core/form-field.model.spec.ts

```diff
@@ -531,18 +531,21 @@ describe('FormFieldModel', () => {
         expect(field.value).toBe(false);
     });
 
-    it('should delete empty dropdown value from the form values', () => {
+    it('should set the value as null for a dropdown field that has the None value selected', () => {
         const form = new FormModel();
         const field = new FormFieldModel(form, {
             id: 'dropdown-1',
             type: FormFieldTypes.DROPDOWN
         });
 
         field.value = 'empty';
-        expect(form.values['dropdown-1']).toBe(undefined);
+        expect(form.values['dropdown-1']).toBe(null);
```

Comment:
why null and not undefined ?

Reply:
Same as https://github.com/Alfresco/alfresco-ng2-components/pull/8021#discussion_r1037042264